### ISSUE TITLE
SuperGrafx support

### ISSUE THREAD

**Issue by mrdeathjr28** [drhelius/Geargrafx#8]
Hi

Maybe can interesting if can add support for Supergrafx - PCEngineCD

Thanks

**Comment by drhelius** [comment]
yes, it's planned

**Comment by Ryusennin** [comment]
Does this only work with the 5 commercial SGX games?

Homebrews don't work, since there doesn't seem to have any option to change the PC Engine model. Maybe GearGrafx could automatically force the SGX mode when running a ROM with an .sgx extension, like Mesen does.

**Comment by drhelius** [comment]
> Does this only work with the 5 commercial SGX games?

Yes you are right. I'm going to add 2 things but didn't have time time yet :

- Force SGX when .sgx extension is found
- Add an option to force SGX in the UI in the same way there is an option to force JAP PCE
